fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -893,6 +895,15 @@ protected JobPayloadBuilder buildJobPayloadBuilder() {
 
 
     /**
+     * Extension point for building a dictionary of base granularities for use in the system.
+     *
+     * @return A map of time grain api name (forced to uppercase) to time grain instances.
```

Comment:
This documentation claims that the timegrain name is forced to uppercase, but in the [lookup logic](https://github.com/yahoo/fili/pull/88/files#diff-b285206fd3b789c0f559db575ccff632R73) we force it to lowercase instead.